docs/design/ovs-pipeline.md

```diff
@@ -1035,7 +1088,13 @@ Nodes, even though `externalTrafficPolicy` is set to `Local` for the Service. Du
 flow 9 exclusively matches packets sourced from the external network, resembling the pattern of flow 1. The target of
 flow 9 is an OVS group that has only the local Endpoints since `externalTrafficPolicy` of the Service is `Local`.
 
-Flow 10 is the table-miss flow.
+Flow 10 and flow 11 are for case 7. Flow 10 is similar to flow 8, while flow 11 diverges slightly from flow 9. The
+additional reg mark `LoadBalancerSourceRangesRegMark`, loaded in table [ServiceMark], is used to match the first
+packet destined for the sample [LoadBalancer with LoadBalancerSourceRanges], originating from a CIDR listed in the
+Service `loadBalancerSourceRanges`. It's worth noting that `loadBalancerSourceRanges` only exclusively influences the
+traffic from the external network as `LoadBalancerSourceRangesRegMark` is not used in flow 10.
```

Comment:
Does this match the kube-proxy behavior? I took a quick look and it seems that even when traffic comes from local Pods, `loadBalancerSourceRanges` is enforced

https://github.com/kubernetes/kubernetes/blob/8119e57c070d21f2a7ff37fa7b02f7f74331e393/pkg/proxy/iptables/proxier_test.go#L2192-L2205

I may be misunderstanding the comment / implementation

Reply:
You are right. Since K8s 1.25, the Pod CIDR is not allowed by default. I have updated the implementation.